### DISCUSSION TITLE
Does anybody have the system description the trainig data is based on?

### DISCUSSION THREAD

**Discussion by ThePokerDude** [lorserker/ben#107] [Q&A]
In B19 of the demo boards it seems BEN does not use 3rd suit forcing but plays two way check back
After 2c BEN bids 2d. In 3rd suit forcing it would bid 2h to show 3 hearts.
In two way check back 2c means - either weak with diamonds or an invitational hand.
I could not find the GIB system description.

![image](https://github.com/lorserker/ben/assets/33804161/5d3d960a-ae44-486c-b2e9-7e451f169eb5)

**Comment by ThorvaldAagaard** [comment]
BEN is currently playing this: http://ben.aalborgdata.dk:8080/

From here you can also play online, when my server is running (most of the time)

The data used for training is more than 25G so even zipped I cannot attach it. I will see if I can make it available in some way.

**Comment by ThorvaldAagaard** [comment]
CC's from BBA is now included in the repository
